configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,9 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read", "write"],
+    "cve.business_risk": ["write"],
+    "cve.status": ["write"],
+    "report": ["read", "generate"],
+    "system": ["exclude", "include", "read"],
+    "remediation": ["create", "update"]
```

Comment:
@blayson remediations has their own permissions / can you check to see if this overlap of having a remediation permission within Vulnerability is necessary?